.github/workflows/create-branch-build.yml

```diff
@@ -58,19 +58,45 @@ jobs:
           BRANCH_ITERATION=${{ github.run_attempt }}.${{ github.run_number }}
           echo "version=$(echo ${{ steps.next-semvers.outputs.patch }}-alpha+BR.${{ github.ref_name }}.${BRANCH_ITERATION})" >> $GITHUB_OUTPUT
 
-  build-test-package:
-    needs: ['set-branch-version']
+  construct-payload:
```

Comment:
Thoughts on making this a matrix input and not duplicating the parsing code for each different payload?

Reply:
Maybe a backlog feature

Reply:
Eh, creating one payload to add to GITHUB_OUTPUT would be tricky I think... so might not be worth it.